app/commands.py

```diff
@@ -832,3 +842,147 @@ def purge_csv_bucket():
     print("ABOUT TO RUN PURGE CSV BUCKET")
     s3.purge_bucket(bucket_name, access_key, secret, region)
     print("RAN PURGE CSV BUCKET")
+
+
+"""
+Commands to load test data into the database for
+Orgs, Services, Users, Jobs, Notifications
+
+faker is used to generate some random fields. All
+database commands were used from tests/app/db.py
+where possible to enable better maintainability.
+"""
+fake = Faker(["en_US"])
+
+
+# generate n number of test orgs into the dev DB
+@notify_command(name="add-test-organizations-to-db")
+@click.option("-g", "--generate", required=True, prompt=True, default=1)
+def add_test_organizations_to_db(generate):
```

Comment:
It looks like this needs a `development/test` environment check here like the other methods:

```suggestion
def add_test_organizations_to_db(generate):
    if getenv("NOTIFY_ENVIRONMENT", "") not in ["development", "test"]:
        current_app.logger.error("Can only be run in development")
        return
```

(We could potentially turn this into a decorator for ourselves here to make it easier, but no need to do so now!)